src/utils/fileTable.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable */
```

Comment:
Try to handle eslint errors rather than disabling it.

Reply:
👍🏻